### PR TITLE
Issue #2313 amend duplicate anchors in documentation

### PR DESCRIPTION
**Pull request by robin-owen**

### CODE REVIEW

File: docs/source/getting-started/preparing-to-install.adoc [hunk 1]
```diff
@@ -1,6 +1,6 @@
 include::variables.adoc[]
 
-[[installing-minishift]]
+[[preparing-install-minishift]]
```
Comment (LalatenduMohanty) [inline docs/source/getting-started/preparing-to-install.adoc]: @robin-owen What about `preparing-minishift-install`

Comment (robin-owen) [reply docs/source/getting-started/preparing-to-install.adoc]: @LalatenduMohanty I'm trying to keep the anchors similar to the titles they refer to. Would you prefer that the title of this section be changed generally?

Comment (LalatenduMohanty) [reply docs/source/getting-started/preparing-to-install.adoc]: @robin-owen Not really, I think I would prefer to keep them similar. Just that it sound grammatically incorrect leaving bad taste in mouth. If you feel strongly that the current form is fine then I can go ahead and merge this.

**Comment by robin-owen** [comment]
@LalatenduMohanty Changed the anchor name.

**Comment by LalatenduMohanty** [comment]
Thanks @robin-owen

**Pull request merged**